dockerfiles/web/Dockerfile

```diff
@@ -31,6 +31,7 @@ COPY package.json /usr/src/app/
 RUN yarn
 
 COPY create_db.py test_data.py /usr/src/app/
+COPY .flake8 /usr/src/app/
```

Comment:
added the flake config to the docker container so I could run flake tests out of it